Fix key names for bond lengths in results table headers
author: Alex Domingo (Vrije Universiteit Brussel)
--- ./host/standalone/adf_new/PyFragModules.py.orig	2024-02-23 13:20:13.952195000 +0100
+++ ./host/standalone/adf_new/PyFragModules.py	2024-02-23 13:20:22.601881593 +0100
@@ -203,8 +203,8 @@
 def WriteTable(tableValues, fileName):
     energyfile = open(f"pyfrag_{fileName}.txt", "w")
     headerlist_all = sorted(tableValues[0])
-    headerlist_select = [e for e in headerlist_all if e not in ("#IRC", "bondlength_1", "EnergyTotal", "Int", "Elstat", "Pauli", "OI", "Disp", "StrainTotal", "frag1Strain", "frag2Strain")]
-    headerlist = ["#IRC", "bondlength_1", "EnergyTotal", "Int", "Elstat", "Pauli", "OI", "Disp", "StrainTotal", "frag1Strain", "frag2Strain"] + headerlist_select
+    headerlist_select = [e for e in headerlist_all if e not in ("#IRC", "bondlength", "EnergyTotal", "Int", "Elstat", "Pauli", "OI", "Disp", "StrainTotal", "frag1Strain", "frag2Strain")]
+    headerlist = ["#IRC", "bondlength", "EnergyTotal", "Int", "Elstat", "Pauli", "OI", "Disp", "StrainTotal", "frag1Strain", "frag2Strain"] + headerlist_select
     writeKey(energyfile, headerlist)
     for entry in tableValues:
         sortedEntry = [entry[i] for i in headerlist]
--- ./host/standalone/adf_newopen/PyFragModules.py.orig	2024-02-23 13:18:00.235407000 +0100
+++ ./host/standalone/adf_newopen/PyFragModules.py	2024-02-23 13:18:49.028392911 +0100
@@ -193,8 +193,8 @@
 def WriteTable(tableValues, fileName):
     energyfile = open(f"pyfrag_{fileName}.txt", "w")
     headerlist_all = sorted(tableValues[0])
-    headerlist_select = [e for e in headerlist_all if e not in ("#IRC", "bondlength_1", "EnergyTotal", "Int", "Elstat", "Pauli", "OI", "Disp", "StrainTotal", "frag1Strain", "frag2Strain")]
-    headerlist = ["#IRC", "bondlength_1", "EnergyTotal", "Int", "Elstat", "Pauli", "OI", "Disp", "StrainTotal", "frag1Strain", "frag2Strain"] + headerlist_select
+    headerlist_select = [e for e in headerlist_all if e not in ("#IRC", "bondlength", "EnergyTotal", "Int", "Elstat", "Pauli", "OI", "Disp", "StrainTotal", "frag1Strain", "frag2Strain")]
+    headerlist = ["#IRC", "bondlength", "EnergyTotal", "Int", "Elstat", "Pauli", "OI", "Disp", "StrainTotal", "frag1Strain", "frag2Strain"] + headerlist_select
     writeKey(energyfile, headerlist)
     for entry in tableValues:
         sortedEntry = [entry[i] for i in headerlist]
